packages/neuron-wallet/src/controllers/anyone-can-pay.ts

```diff
@@ -0,0 +1,93 @@
+import AssetAccountInfo from 'models/asset-account-info'
+import Transaction from "models/chain/transaction"
+import { ServiceHasNoResponse } from "exceptions"
+import { ResponseCode } from "utils/const"
+import AnyoneCanPayService from "services/anyone-can-pay"
+import TransactionSender from "services/transaction-sender"
+import { set as setDescription } from 'database/leveldb/transaction-description'
+
+export interface GenerateAnyoneCanPayTxParams {
+  walletID: string
+  address: string
+  amount: string
+  assetAccountID: number
+  feeRate: string
+  fee: string
+  description?: string
+}
+
+export interface GenerateAnyoneCanPayAllTxParams {
+  walletID: string
+  address: string
+  assetAccountID: number
+  feeRate: string
+  fee: string
+  description?: string
+}
+
+export interface SendAnyoneCanPayTxParams {
+  walletID: string
+  tx: Transaction
+  password: string
+}
+
+export default class AnyoneCanPayController {
+  public async generateTx(params: GenerateAnyoneCanPayTxParams): Promise<Controller.Response<Transaction>> {
+    const tx = await AnyoneCanPayService.generateAnyoneCanPayTx(
+      params.walletID,
+      params.address,
+      params.amount,
```

Comment:
I couldn't find the logics to handle this `amount` variable in the `AnyoneCanPayService`: https://github.com/nervosnetwork/neuron/blob/c7e3067fd0864d82724139de73751764ce56c337/packages/neuron-wallet/src/services/anyone-can-pay.ts#L19